package/MDAnalysis/coordinates/base.py

```diff
@@ -1132,12 +1137,7 @@ def _read_next_timestep(self, ts=None):  # pragma: no cover
 
     def __iter__(self):
         self._reopen()
```

Comment:
Closing and opening in causes flushes to the filesystem which are bad if you do a few hundreds a second. Since looping usually takes a while I'm good with reopening trajectories. In not sure we supported rewind for all readers.